README.md

```diff
@@ -105,6 +105,64 @@ const tracer = opentracing.globalTracer();
 
 Note: `globalTracer()` returns a wrapper on the actual tracer object. This is done for the convenience of use as it ensures that the function will always return a non-null object.  This can be helpful in cases where it is difficult or impossible to know precisely when `initGlobalTracer` is called (for example, when writing a utility library that does not control the initialization process).  For more precise control, individual `Tracer` objects can be used instead of the global tracer.
 
+### Span manager (experimental)
+
+This library supports span activation in the spirit of the [Scope Manager](https://github.com/opentracing/specification/blob/f7ca62c9/rfc/scope_manager.md) draft RFC.
+
+If neither `childOf` nor `references` is specified, create spans are children of the activated span.
+
+The intent of `SpanManager` is that it remains active within the executed function and its transitive call graph. Ideally, this applies to both synchronous and asynchronous calls.
+
+```js
+tracer.activeSpan() // null
+tracer.activate(span, () => {
+    tracer.activeSpan() // span
+    setTimeout(() => {
+        tracer.activeSpan() // span
+    }, 0);
+    (async () => {
+        tracer.activeSpan() // span
+    })();
+});
+tracer.activeSpan() // null
+```
+
+JavaScript support for continuation local storage is a patchwork.
+This library includes simple wrappers around a few common APIs for continuation contexts.
+
+**ZoneSpanManger**
+
+This uses [Zone.js](https://github.com/angular/zone.js). (See the associated [Zone](https://github.com/domenic/zones) ES TC39 proposal.)
+It includes support for most common APIs: Web, Node.js, Electron, RxJS. Async syntax (async/await) is not supported.
+I.e. you should target ES 2015 or earlier syntax.
+
+```js
+import 'zone.js';
+import { ZoneSpanManager } from 'opentracing/zone_span_manager';
+
+new ZoneSpanManager();
+```
+
+**AsyncHookSpanManager**
+
+This uses the Node.js [async_hooks](https://nodejs.org/api/async_hooks.html) API, available in Node.js 8.1.x+.
+
+```js
+import { AsyncHookSpanManager } from 'opentracing/async_hook_span_manager';
+
+new AsyncHookSpanManager();
+```
+
+**AsyncWrapSpanManager**
```

Comment:
Since `AsyncWrap` cannot have multiple hooks, and has many bugs, possible crashes and memory leaks in older Node versions, I would probably skip it entirely and add something based on a fixed fork of `async-listener` as described in https://github.com/opentracing/opentracing-javascript/issues/112#issuecomment-433982906

Reply:
FWIW, in my experience it's also slower than `async-listener`.